T/Tcl/build_tarballs.jl

```diff
@@ -36,7 +38,7 @@ platforms = supported_platforms()
 
 # The products that we will ensure are always built
 products = [
-    LibraryProduct(["libtcl8.6", "libtcl8", "tcl86"], :libtcl86),
+    LibraryProduct(["libtcl8.6", "libtcl8", "tcl86"], :libtcl),
```

Comment:
On a second thought, I decided that we shouldn't encode the version number in the variable name.  What do you thing?

Reply:
Strongly agree.